### ISSUE TITLE
--options params don't work with arguments required

### ISSUE THREAD

**Issue by scorsi** [squareboat/nest-console#3]
Hello, 
Here's a minimal reproduction Command:
```typescript
@Injectable()
export class AppService {
  @Command('organization:create', {
    desc: 'Create an organization',
    args: { name: { req: true, desc: 'The name of the organization' } },
  })
  createOrganization(args: CommandArguments) {
    return;
  }
}
```

If you try `node cli organization:create --options` you will found `Missing arguments: name` error.

`--options` should not check for required params.

Thanks,

**Comment by vinayak25** [comment]
Thank you reporting this, If you were able to find the issue please raise a PR, or I will pick it and fix it :)

**Comment by scorsi** [comment]
I already take a look at the source code, the check for the options argument is simply done after the one for required arguments.
Will work on it tomorow to create and submit a PR.

**Comment by vinayak25** [comment]
Released in [`v0.0.4`](https://github.com/squareboat/nest-console/releases/tag/v0.0.4)
